src/service/extension-location.js

```diff
@@ -105,10 +115,37 @@ export function calculateEntryPointScriptUrl(
 export function parseExtensionUrl(scriptUrl) {
   // Note that the "(\.max)?" group only applies to local dev.
   const matches = scriptUrl.match(
-    /^(.*)\/(.*)-([0-9.]+|latest)(\.max)?\.(?:js|mjs)$/i
+    /^(.*)\/(.*)-([0-9.]+|latest)(\.max)?\.(?:js|mjs)(\?.*)?$/i
```

Comment:
So this is actually the regex we'd have to change to support `.sxg.js`. If we're doing this, do we need to go through the extra effort to support both `.sxg.js` and `?f=sxg`?

Reply:
I think we would have to do it, because on local dev we make do with `.sxg.js`. In prod, we'd have to match `.js?f=sxg` and `.mjs?f=sxg`.

Reply:
this can still be handled by the local web server. this should be straightforward in `gulp serve`, this is a lot more difficult with karma and its test server

Reply:
Please take a look at the regex change I just made.